CMakeLists.txt

```diff
@@ -1,15 +1,23 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.19)
+# CMake compatibility issues: don't modify this, please!
+CMAKE_MINIMUM_REQUIRED( VERSION 2.4.6 )

```

Comment:
Why do you want to go back ?

Reply:
Looks like it failed to build otherwise?
https://github.com/kevinbentley/Descent3/actions/runs/8742874968/job/23992131506